File: app/src/main/java/de/impulse/spieleabend/MainActivity.java
package de.impulse.spieleabend;

import android.os.Bundle;
import android.view.Gravity;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        TextView helloWorld = new TextView(this);
        helloWorld.setGravity(Gravity.CENTER);
        helloWorld.setText("Hello World!");
        helloWorld.setTextSize(24);

        setContentView(helloWorld);
    }
}

File: app/src/main/java/de/impulse/spieleabend/MainActivity.kt
package de.impulse.spieleabend

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.SystemBarStyle
import androidx.activity.compose.setContent
import androidx.activity.enableEdgeToEdge
import dagger.hilt.android.AndroidEntryPoint
import de.impulse.spieleabend.frontend.navigation.SpieleabendApp
import de.impulse.spieleabend.frontend.theme.SpieleabendTheme

@AndroidEntryPoint
class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        enableEdgeToEdge(
            statusBarStyle = SystemBarStyle.dark(android.graphics.Color.TRANSPARENT),
            navigationBarStyle = SystemBarStyle.dark(DARK_SYSTEM_BAR_COLOR),
        )

        setContent {
            SpieleabendTheme {
                SpieleabendApp()
            }
        }
    }

    private companion object {
        const val DARK_SYSTEM_BAR_COLOR = 0xFF0C0F15.toInt()
    }
}
